lib/serializers/resource-serializer.js

```diff
@@ -273,7 +274,7 @@ class ResourceSerializer extends EsSerializer {
           .map(ResourceSerializer.applySortableShelfMark)
           .sort((i1, i2) => { return i1.shelfMark_sort > i2.shelfMark_sort ? 1 : -1 })
 
-        this.addStatement('numItems', itemsSerialization.length)
+        this.addStatement('numItems', itemsSerialization.length - numElectronicResources)
```

Comment:
`numElectronicResources` really represents the number of URLs found in all the items (i.e. a single "electronic item" can have many electronicLocator URLs). So by subtracting `numElectronicResources` from the number of items, you may subtract too many. Really, you should only be subtracting the number of items that have URLs in them. So a fix is:
```
const numItemsWithUrls = this.object._items
  .filter((item) => item.has('bf:electronicLocator'))
  .length
this.addStatement('numItems', itemsSerialization.length - numItemsWithUrls)
```